### PR TITLE
Feature/user class tdd

### PR DESCRIPTION
**Pull request by dreardon1021**
#### What's this PR do?
Creates a file for user TDD
#### Where should the reviewer start?
Start by creating a class file for user. Then export the User class. Run tests to make sure testing file paths/requires/exports are all communicating correctly.
#### How should this be manually tested?
Create the user class and complete the TDD tests in user-test.js
#### Questions:
- Unsure if file paths/requires are operating correctly

### CODE REVIEW

File: test/users-test.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+describe('User', function() {
+  beforeEach(function() {
+    const user = new User(userData[0]);
+  });
+
+  it('should be a function', function() {
+    expect(user).to.be.a('function');
+  });
+
+  it('should be an instance of Turn', function() {
```
Comment (jonathan-tschida) [inline test/users-test.js]: Turn? I think this should say User

File: test/users-test.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const userData = data.testData;
```
Comment (jonathan-tschida) [inline test/users-test.js:5]: I think you need a line above this thats something like `const data = require('./data/users.js');`

File: test/data/users.js [hunk 3]
```diff
@@ -27,3 +27,11 @@ const testData = [
     "friends": []
   }
 ];
+
+// module.exports = {
+//   testData
+// }
+
+if (typeof module !== 'undefined') {
+   module.exports = testData;
```
Comment (jonathan-tschida) [inline test/data/users.js]: I think testData needs to be wrapped in {} to make it an object so we can access the properties

**Comment by jonathan-tschida** [comment]
If you fix a few things and add skips to the tests then I will merge it

File: test/data/users.js (hunk 3 above)
Comment (dreardon1021) [reply test/data/users.js]: I believe it is. It is currently setup how you do and things are working

File: test/users-test.js (hunk 2 above)
Comment (dreardon1021) [reply test/users-test.js:5]: Will add this

File: test/users-test.js (hunk 1 above)
Comment (dreardon1021) [reply test/users-test.js]: Corrected.

File: test/data/users.js (hunk 3 above)
Comment (dreardon1021) [reply test/data/users.js]: I went in to the resolve conflicts area and went with what we had on the master as it was working properly

**Comment by dreardon1021** [comment]
Skips added;
Resolved merge conflict in users.js for the export;
added 'let user ' so the beforeEach works properly
added require data

**Pull request merged**